function/values.yaml

```diff
@@ -0,0 +1,16 @@
+replicas: 0
+useInterpodAntiAffinity: false
+memoryRequests: 256Mi
+cpuRequests: 250m
+memoryLimits: 1Gi
+cpuLimits: 1
+imagePullPolicy: IfNotPresent
+dnsConfig:
```

Comment:
shouldn't this be in the lib chart?

Reply:
I don't think values in library chart will be used when using templates. That was my understanding , can try if you want . Also wanted to have empty values in library to avoid confusion on who is overriding what.

Reply:
I thought they were used, would be nice to try, not a blocker

Reply:
Tried this, library values are ignored.